soh/soh/Enhancements/mods.cpp

```diff
@@ -486,6 +487,159 @@ void RegisterBonkDamage() {
     });
 }
 
+void RegisterChestapalooza() {
+    srand((unsigned)time(NULL));
+    static int roll;
+    static int statusTimer = -1;
+    static int eventTimer = -1;
```

Comment:
Is there ever a check for these to be -1 later on, or are they only checked for 0? If the latter, then you'd probably want to change them to uint32_t. Also, I think someone else would come along and tell you to specifically use the `_t` definitions of numerical types, so this would be int32_t if you need the -1.

Reply:
so when i initially tested it with 0 start the code that executes the explosions and such (which needed a delay) would fire off immediately, so i set it -1 to avoid that, it will only ever be 0 (which fires off the trap) when set by the case statement.

I can update these to the _t variant no problem.